contrib/coredns/policy/client.go

```diff
@@ -9,6 +9,16 @@ import (
 	"github.com/infobloxopen/themis/pep"
 )
 
+type pepCacheHitHandler struct{}
+
+func (ch *pepCacheHitHandler) Handle(req interface{}, resp interface{}, err error) {
+	log.Printf("[INFO] PEP responding to PDP request from cache %+v", req)
```

Comment:
This line breaks master. Should be log.Infof()

Reply:
Yes, that was my bad. 

I've already submitted PR #207 to fix that. PTAL. 

Thanks!